inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/conv_bias.cpp

```diff
@@ -0,0 +1,150 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/base/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+
+namespace SubgraphTestsDefinitions {
+
+using ConvBiasTestParams = std::tuple<nodeType,
+                                      SizeVector,  // input shape
+                                      SizeVector>; // kernel size
+
+class ConvBiasTest : public testing::WithParamInterface<ConvBiasTestParams>, virtual public LayerTestsUtils::LayerTestsCommon {
```

Comment:
CPU specific Convolution test supports test cases with post ops. Why we need separate one?

Reply:
existing tests was updated